05_stringer/brvishnu92/main_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//main() test
+func TestMainOutput(t *testing.T) {
```

Comment:
Again here I don't see the benefit in this comment. The name of the function is documenting the same thing.

Reply:
Done.